docs/manual/guides/install-test-versions.de.md

```diff
@@ -50,18 +50,18 @@ werden.
 ## Release Candidates installieren
 
 Release Candidates nutzen eine spezifische Form von sogenannten »Release-Tags«.
-Der erste Release Candidate von Contao `5.5` heißt z. B. `5.5.0-RC1`. Normalerweise
-würde Composer nur _stabile_ Versionen installieren und daher solche Release Candidates
-nicht beachten.
+Der erste Release Candidate von Contao `{{% siteparam "currentReleaseCandidate" %}}` heißt z. B.
+`{{% siteparam "currentReleaseCandidate" %}}.0-RC1`. Normalerweise würde Composer nur _stabile_ Versionen installieren
+und daher solche Release Candidates nicht beachten.
 
 Um die Installation von Release Candidates zu erlauben, muss die `minimum-stability` auf `RC` runter gesetzt werden.
 Alternativ kann man über sogenannte [stability-flags][ComposerStabilityContraints] direkt bei einzelnen Paketen
-niedrigere Stabilitäten erlauben, z. B. mit `"contao/manager-bundle": "5.5.*@RC"` in diesem Fall. In diesem Fall muss man
-allerdings auch zusätzlich das `contao/core-bundle` in die `composer.json` mit aufnehmen, um auch dediziert die
-niedrigere Stabilität bei diesem Paket zu erlauben.
+niedrigere Stabilitäten erlauben, z. B. mit `"contao/manager-bundle": "{{% siteparam "currentReleaseCandidate" %}}.*@RC"`
+in diesem Fall. In diesem Fall muss man allerdings auch zusätzlich das `contao/core-bundle` in die `composer.json` mit
```

Comment:
```suggestion
. In diesem Fall muss man allerdings auch zusätzlich das `contao/core-bundle` in die `composer.json` mit
```

Reply:
Would be unrelated to the PR but that makes sense to fix.